.github/workflows/CI.yml

```diff
@@ -21,7 +23,8 @@ jobs:
           - '1.8'
           - '1.9'
           - '1.10'
-          - '^1.11.0-beta2'
+          - '1.11'
+          - 'nightly'
```

Comment:
Is nightly the right thing here? Do you usually use something more specific on 1.12.0-*?